### PR TITLE
Remove facebook login option

### LINKED ISSUE #269: For Nathan: Fix Facebook login on the event platform
When someone wants to book a ticket, they need to sign up on the event platform. There are currently three options but the FB is not working. Can you please fix?
image
image

### PR DESCRIPTION
**Pull request by pro-Nate**

### CODE REVIEW

File: js/src/auth/Login.js [hunk 1]
```diff
@@ -143,20 +143,23 @@ class Login extends React.Component {
...
+        <Row className="justify-content-center">
+          <Col xl="4" lg="6" md="8" className="text-center my-1">
             <Button onClick={this.google_auth.bind(this)} color="primary">
               <FontAwesomeIcon icon={['fab', 'google']} className="mr-2"/>
               Login with Google
             </Button>
+            <hr className="mt-4 mb-0"/>
           </Col>
-          <Col lg="3" md="4" className="text-center text-md-right my-1">
+          {/* <Col lg="3" md="4" className="text-center text-md-right my-1">
```
Comment (samuelcolvin) [inline js/src/auth/Login.js:155]: best to remove this properly, commenting it out will only lead to confusion down the line.

And below.

**Review by samuelcolvin — CHANGES_REQUESTED** [review]
you also need to remove the endpoints serverside.

**Comment by samuelcolvin** [comment]
Sorry to be a pedant, please don't include the issue ID in the title, it's not clickable and looks confusing.

**Comment by pro-Nate** [comment]
> you also need to remove the endpoints serverside.

Ok will do. I wasn't 100% sure they wanted this as a final solution but will remove fully.

**Comment by samuelcolvin** [comment]
I think best to just rip it out fully.

It would need significantly rewriting if we wanted to use it again, and I don't think anyone clicks it.

**Comment by pro-Nate** [comment]
> I think best to just rip it out fully.
> 
> It would need significantly rewriting if we wanted to use it again, and I don't think anyone clicks it.

I've removed the client side facebook code. Decided to commit the server side changes in a different PR

**Pull request merged**